src/uct/ib/base/ib_log.c

```diff
@@ -154,7 +163,7 @@ static void uct_ib_dump_wr(struct ibv_qp *qp, uct_ib_opcode_t *op,
 }
 
 static void uct_ib_dump_send_wr(uct_ib_iface_t *iface, struct ibv_qp *qp,
-                                struct ibv_send_wr *wr,
+                                struct ibv_send_wr *wr, uct_am_trace_type_t am_type,
```

Comment:
maybe pass the max number of SG elements to dump?
short,bcopy: inf
zcopy: 1 if gdr enabled, inf otherwise (save uct_ib_iface_device(iface)->max_zcopy_log_sge)